packages/react/src/primitives/types/autocomplete.ts

```diff
@@ -143,6 +143,4 @@ export interface UseAutocompleteProps extends Partial<AutocompleteProps> {
   onChange: React.ChangeEventHandler<HTMLInputElement>;
 
   onClick: React.MouseEventHandler<HTMLInputElement>;
-
-  onFocus: React.FocusEventHandler<HTMLInputElement>;
```

Comment:
This is internal API

Reply:
Does that mean this is a breaking change 😢

Reply:
Nope, the interface is only used internally

Reply:
Thanks for adding the edit